datagetter.py: changed default branch of schema

mrshll1001 opened the pull request:
Changed the default branch of the schema in datagetter.py to 'main' from
'master' to reflect upcoming change in the schema repo

Bjwebb commented:
(This was done as part of this maintenance trello card https://trello.com/c/XROQvffQ/54-pip-resolver-changing.)

michaelwood commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Refactor cove-360 into its own application](https://trello.com/c/53ruAkzf/19-refactor-cove-360-into-its-own-application)

Bjwebb commented:
This has been fixed with the latest requirements updates.

mrshll1001 committed:
datagetter.py: changed default branch of schema

Changed the default branch of the schema in datagetter.py to 'main' from
'master' to reflect upcoming change in the schema repo

michaelwood commented:
Thanks, a rebased version of this has been merged
